voter-dapp/src/TokenTracker.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import { drizzleReactHooks } from "@umaprotocol/react-plugin";
+import { useTableStyles } from "./Styles.js";
+
+function TokenTracker() {
+  const { drizzle } = drizzleReactHooks.useDrizzle();
+  const classes = useTableStyles();
+
+  const addTokenToWallet = () => {
+    window.web3.currentProvider.sendAsync({
+      method: "wallet_watchAsset",
+      params: {
+        type: "ERC20",
+        options: {
+          address: drizzle.contracts.VotingToken.address,
+          symbol: "UMA",
+          decimals: 18,
+          image: "https://umaproject.org/assets/images/UMA_square_red_logo.png"
```

Comment:
@rcai1 @allisonlu22 do we want the UMA logo to show up for people's tokenbuilder tokens...?

Reply:
My preference is for the UMA logo itself to be reserved for the UMA token, rather than synthetic tokens that are created via the Synthetic Token Builder. 

We could create an alternative logo for synthetic tokens. Let me know if you'd like me to make a few mock-ups.

Reply:
Agreed, @rcai1 and @chrismaree, y'all can decide what logo (if any) to use here.

Reply:
I spent some time playing with different colors. 
![UMA Token Icons options](https://user-images.githubusercontent.com/12886084/73769693-16dd2d80-4749-11ea-820b-906549934e8e.png)